.bazelrc

```diff
@@ -144,21 +144,23 @@ build:osx --noincompatible_run_shell_command_string
 
 # Debug mode.
 build:debug -c dbg
-build:debug --copt='-O3'
-build:debug --copt='-march=native'
+
+# On Windows, we always generate debug info, and always compile with
+# optimisation flags.
+build:windows --config=release
+build:windows --copt='/Z7' --copt='/UNDEBUG'
+# Disable inlining of functions not marked as "inline".
+build:windows --copt='/Ob1'
+build:windows --linkopt='/DEBUG:FULL'
 
 # Release mode.
 build:release -c opt
-build:release --copt='-O3'
-build:release --copt='-march=native'
-build:release --copt='-DNDEBUG'
 # NDEBUG disables asserts.
```

Comment:
I think this comment belonged to the line you removed?

Reply:
-c opt enables NDEBUG (which is annoying btw). It belongs to the next line, which is now commented out, because I can't do "build:release:gnulike".